src/Gofer-Tests/GoferApiTest.class.st

```diff
@@ -141,13 +154,15 @@ GoferApiTest >> testPackageReference [
 	gofer 
 		repository: self monticelloRepository;
 		package: 'GoferFoo'.
-	self assert: (gofer resolved size = 1).
-	self assert: (gofer resolved first isKindOf: GoferResolvedReference).
-	self assert: (gofer resolved first packageName = 'GoferFoo').
-	self assert: (gofer resolved first author = 'lr').
-	self assert: (gofer resolved first branch isEmpty).
-	self assert: (gofer resolved first versionNumber = 4).
-	self assert: (gofer resolved first repository = self monticelloRepository)
+
+	self 
+		assert: gofer resolved size equals: 1;
+		assert: (gofer resolved first isKindOf: GoferResolvedReference);
+		assert: gofer resolved first packageName equals: 'GoferFoo';
```

Comment:
Could be cool to do the same as in testConstraintReference by saving the reference.